.github/CONTRIBUTING.md

```diff
@@ -17,9 +17,23 @@ git config user.email user@example.com
 You can generate the IntelliJ project by running
 ```
 ./gradlew idea       # *nix-based systems
-./gradlew.bat idea   # Windows
+gradlew idea         # Windows
 ```
-then "Open" it with IntelliJ. Unfortunately, Eclipse project import isn't straightforward so it's not recommended.
+then "Open" it with IntelliJ.
+
+You can generate the Eclipse projects by running
+```
+./gradlew eclipse   # *nix-based systems
```

Comment:
Have you found this to work in practice? My understanding was that this doesn't currently do the right thing for the gradle/gradle build.